foursight_core/fs_connection.py

```diff
@@ -46,18 +46,20 @@ def __init__(self, fs_environ, fs_environ_info, test=False, use_es=True, host=No
         self.ff_env = fs_environ_info['ff_env']
         self.ff_es = fs_environ_info['es']
         self.ff_bucket = fs_environ_info['bucket']
+        self.redis = None
+        self.redis_url = None
         try:
             if 'redis' in fs_environ_info:
-                self.redis = RedisBase(create_redis_client(url=fs_environ_info['redis']))
+                self.redis_url = fs_environ_info['redis']
+                self.redis = RedisBase(create_redis_client(url=self.redis_url))
             elif 'REDIS_HOST' in os.environ:  # temporary patch in until env config is fully sorted - Will
-                self.redis = RedisBase(create_redis_client(url=os.environ['REDIS_HOST']))
-            else:
-                self.redis = None
+                self.redis_url = os.environ['REDIS_HOST']
+                self.redis = RedisBase(create_redis_client(url=self.redis_url))
         except redis.exceptions.ConnectionError:
```

Comment:
I am not a huge fan of really narrow error catching here. I know the common wisdom is to be specific, but if the system is prepared to run without Redis, do you really want to crash here? What would be the harm in catching Exception?

Reply:
But it isn't crashing/exceptioning here, it catches, prints error, and moves on; am I missing something?